### PR TITLE
Add filter to MySquads so that only the current user's squads will show up

### PR DESCRIPTION
**Pull request by tfettig22**
## What does this PR do?
- [x] New feature (adds functionality)
- [x] Debugging/Refactor
 
### Where should your reviewer start?
- MySquads screen
### What (if any) features are you implementing?
- Adds a filter function so that only the current user's squads show up on the squads page
### What (if anything) did you refactor?
- setting the state of MySquads
### What issues does this PR close?
- 
### Is there anything that you need from your teammates?
-
### Any other comments, questions, or concerns?
-
### How Has This Been Tested?
- [x] I tested my changes in the browser
- [] I tested with Cypress

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [x] My changes generate no new bugs
- [] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/MySquads.js [hunk 1]
```diff
@@ -37,7 +37,10 @@ const MySquads = ({ userID }) => {
     React.useCallback(() => {
       getUserSquad(userID)
         .then(({ data }) => {
-          const squads = data.map((attribute) => {
+          const filteredSquads = data.filter(squad => {
+            return squad.attributes.squad.members.some(member => member.id.toString() === userID.toString())
+          })
+          const squads = filteredSquads.map((attribute) => {
```
Comment (NickLiggett) [inline src/components/MySquads.js:43]: Big brain!

**Review by NickLiggett — APPROVED** [review]
Nice work, Tom!

File: App.js [hunk 2]
```diff
@@ -16,15 +16,17 @@ const Drawer = createDrawerNavigator();
...
 const App = () => {
   const [userGames, setUserGames] = useState([]);
-  const [user, setUser] = useState({});
+  const [currentUser, setCurrentUser] = useState({});
   const [allUsers, setAllUsers] = useState([]);
 
   useEffect(() => {
-    getSingleUser(1).then(data => {
-      setUser(data.data)
+    getSingleUser(6).then(data => {
```
Comment (aspitz1) [inline App.js]:
```suggestion
    getSingleUser(1).then(data => {
```

**Review by aspitz1 — COMMENTED** [review]
Looks good Tom, pulled it down and it is working

**Pull request merged**